clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -295,7 +296,7 @@ public void onFailure(RuntimeException e) {
      * be polled to see if the request completed successfully.
      * @return A request future whose completion indicates the result of the JoinGroup request.
      */
-    private RequestFuture<ByteBuffer> sendJoinGroupRequest() {
+    private RequestFuture<ByteBuffer> performGroupJoin() {
```

Comment:
Why do we want to change this function name? I feel the original function name is fine. Anyways, if we do want to change it we need to change the comments as well.